tests/kernel_bundle/sycl_join_check_kernel_execution.cpp

```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests to check kernel execution after sycl::join
+//  (State == executable only)
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "../common/get_cts_string.h"
+#include "sycl_join.h"
+
+#include <algorithm>
+
+#define TEST_NAME sycl_join_check_kernel_execution
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+using namespace sycl_cts::tests::sycl_join;
+
+struct kb_join_post_exe_kernel;
+
+/** @brief Execute given kernel and check that the kernel produces correct
+ *         results
+ *  @tparam KernelName Name of kernel to check
+ */
+template <typename KernelName>
+void run_verification(util::logger &log, sycl::queue &q,
+                      sycl::kernel_bundle<sycl::bundle_state::executable> &kb) {
+  bool value_to_rewrite_in_kernel = false;
+  {
+    sycl::buffer<bool, 1> res_buffer(&value_to_rewrite_in_kernel,
+                                     sycl::range<1>(1));
+    q.submit([&](sycl::handler &cgh) {
+      auto res_acc =
+          res_buffer.template get_access<sycl::access_mode::write>(cgh);
+      cgh.use_kernel_bundle(kb);
+      cgh.single_task<KernelName>([=]() { res_acc[0] = true; });
```

Comment:
```suggestion
      cgh.single_task<KernelName>([=] { res_acc[0] = true; });
```

Reply:
I've applied this comment [here](https://github.com/KhronosGroup/SYCL-CTS/pull/229/commits/05b37d37e6f10c1afd5ea287aaa95be0d52d9259#diff-baef877460d6a2ce4e330d5cd9c11f26a0d498776535ee294b1f30540ce86a89R39)